specification/confidentialledger/Microsoft.CodeTransparency/main.tsp

```diff
@@ -28,10 +27,793 @@ namespace Microsoft.CodeTransparency;
+@get
+op getPublicKeys is Foundations.Operation<
+  {},
+  JwksDocument,
+  ServiceTraits,
+  JsonServerErrors
+>;
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@removed(Versions.`2025-01-31-preview`)
+@doc("Get service parameters")
+@route("/parameters")
+@get
+op getParameters is Foundations.Operation<{}, ParametersResult, ServiceTraits>;
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@removed(Versions.`2025-01-31-preview`)
+@doc("Get the DID configuration file.")
+@route("/.well-known/did.json")
+@get
+op getDidConfig is Foundations.Operation<{}, DidDocument, ServiceTraits>;
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@doc("Get CodeTransparency configuration information.")
+@route("/configuration")
+@get
+op getCodeTransparencyConfig is Foundations.Operation<
+  {},
+  CodeTransparencyConfiguration,
+  ServiceTraits,
+  JsonServerErrors
+>;
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@doc("Get CodeTransparency version information.")
+@route("/version")
+@get
+op getCodeTransparencyVersion is Foundations.Operation<
+  {},
+  VersionResult,
+  ServiceTraits,
+  JsonServerErrors
+>;
+
+// -----------------------------------------------------------------
+// Service entry endpoints
+// -----------------------------------------------------------------
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@removed(Versions.`2025-01-31-preview`)
+@doc("Post an entry to be registered on the CodeTransparency instance.")
+@post
+@sharedRoute
+@route("/entries")
+op createEntry is Foundations.Operation<
+  SignedStatement,
+  CreateEntryResult,
+  ServiceTraits
+>;
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@added(Versions.`2025-01-31-preview`)
+@doc("Post an entry to be registered on the CodeTransparency instance, see IETF SCITT SCRAPI draft")
+@post
+@sharedRoute
+@route("/entries")
+op createEntryChanged is Foundations.Operation<
+  SignedStatement,
+  CreateEntryCreated | CreateEntryPending,
+  ServiceTraits,
+  AnyCborError
+>;
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@removed(Versions.`2025-01-31-preview`)
+@doc("Get the status the entry operation")
+@route("/operations/{operationId}")
+op getEntryStatus is Foundations.Operation<
+  OperationIdParameter,
+  GetOperationResult,
+  ServiceTraits
+>;
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-names" "Existing name"
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@removed(Versions.`2025-01-31-preview`)
+@doc("Get the status of all entries operations")
+@route("/operations")
+op listEntryStatuses is Foundations.Operation<
+  {},
+  ListOperationResult,
+  ServiceTraits
+>;
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@removed(Versions.`2025-01-31-preview`)
+@doc("Get Entry with or without receipt embedded in its unprotected header")
```

Comment:
Do we want to clarify that Entries are always Signed Statements?

Reply:
changed to entry -> signed statement